frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -87,6 +98,30 @@ export const StaticEmbeddingOptionCard = () => {
   );
 };
 
+export function EmbeddingSdkOptionCard() {
```

Comment:
This is the new SDK card
![image](https://github.com/user-attachments/assets/aa20ea5b-a96b-4604-8eb1-825a429a4cbc)